packages/nimble-components/package.json

```
@@ -25,15 +25,15 @@
     "generate-scss:bundle": "rollup --bundleConfigAsCjs --config build/generate-scss/rollup.config.js",
     "generate-scss:run": "node build/generate-scss/dist/index.js",
     "tdd": "npm run build-components && npm run test-chrome",
-    "tdd:watch": "npm run build-components:watch & npm run test-chrome:watch",
-    "test-chrome:debugger": "karma start karma.conf.js --browsers=ChromeDebugging",
-    "test-chrome:verbose": "karma start karma.conf.verbose.js --browsers=ChromeHeadlessOpt --single-run",
-    "test-chrome:watch": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --watch-extensions js",
-    "test-chrome": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --single-run",
-    "test-firefox": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run",
-    "test-webkit": "karma start karma.conf.js --browsers=WebkitHeadless --single-run",
-    "test-webkit:debugger": "karma start karma.conf.js --browsers=Webkit",
-    "test": "npm run test-chrome:verbose"
+    "tdd:watch": "npm run build-components:watch && npm run test-chrome:watch",
```

The single `&` that was used here was intentional. The single `&` runs commands in parallel instead of sequential. That is being done so the two watch commands execute simultaneously

```suggestion
    "tdd:watch": "npm run build-components:watch & npm run test-chrome:watch",
```